### PR TITLE
frontend/transport: print Grafana specific headers in slow query log

### PR DESCRIPTION
**Pull request by GiedriusS**
Print some Grafana specific headers in the slow query log if they exist. See this [pull request](https://github.com/grafana/grafana/pull/60301) for more info.

Signed-off-by: Giedrius Statkevičius <giedrius.statkevicius@vinted.com>

### CODE REVIEW

File: pkg/frontend/transport/handler.go [hunk 1]
```diff
@@ -174,12 +174,23 @@ func (f *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
...
 func (f *Handler) reportSlowQuery(r *http.Request, queryString url.Values, queryResponseTime time.Duration) {
+	// NOTE(GiedriusS): see https://github.com/grafana/grafana/pull/60301 for more info.
+	grafanaDashboardUID := "-"
+	if dashboardUID := r.Header.Get("X-Dashboard-Uid"); dashboardUID != "" {
+		grafanaDashboardUID = dashboardUID
+	}
+	grafanaPanelID := "-"
+	if panelID := r.Header.Get("X-Panel-Id"); panelID != "" {
+		grafanaPanelID = panelID
+	}
```
Comment (yeya24) [inline pkg/frontend/transport/handler.go:185]: Looks good. What about having this in the `reportQueryStats`, too?

Comment (GiedriusS) [reply pkg/frontend/transport/handler.go:183]: @yeya24 good idea, added!

**Comment by yeya24** [comment]
Looks like the unit test failures are related. Can you please take a look?

**Comment by GiedriusS** [comment]
@yeya24 thanks, should be good now :bow:

**Review by yeya24 — APPROVED** [review]
Thanks

**Review by alanprot — APPROVED** [review]

**Pull request merged**